TransactionMobile/TransactionMobile/App.xaml.cs

```diff
@@ -45,6 +47,11 @@ public partial class App : Application
         /// </summary>
         public static Guid MerchantId;
 
+        /// <summary>
+        /// The contract products
+        /// </summary>
+        public static List<ContractProductModel> ContractProducts;
```

Comment:
Codacy found an issue: [Change the visibility of 'ContractProducts' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/stuart_ferguson1/MobileApplication/pullRequest?prid=5918284)